content/gettingstarted/index.adoc

```diff
@@ -16,129 +16,132 @@ toc::[]
 
-If OpenShift is installed locally on the machine you are using, the following command should log you in as a user with the `cluster-admin` role:
+==== OpenShift Preparation
+
+If you are installing on OpenShift, you need to grant the `cluster-admin` role to the user that is installing Istio and Kiali.  If OpenShift is installed locally on the machine you are using, the following command should log you in as user `system:admin` which has this `cluster-admin` role:
 
 [source,bash]
 ----
 oc login -u system:admin
 ----
 
-You can now install Istio if needed. See the link:https://istio.io/docs/setup/[Istio Setup docs] for details.
-
-=== Install Kiali
-
-Now install the latest release of Kiali.
+==== Kubernetes Preparation
 
-icon:lightbulb[size=1x]{nbsp} The install script can be configured for a customized install by setting environment variables when running the script. For example, to install a different version of Kiali, set IMAGE_VERSION to the version you want. Also, if your Jaeger or Grafana are accessible at different URLs than the default, then set JAEGER_URL and GRAFANA_URL appropriately. The defaults are: `JAEGER_URL="http://jaeger-query-istio-system.127.0.0.1.nip.io"` and `GRAFANA_URL="http://grafana-istio-system.127.0.0.1.nip.io"`
-
-icon:bullhorn[size=1x]{nbsp} This script requires `envsubst` installed and in your PATH; you can get it via the GNU `gettext` package.
+If you are not installing on OpenShift but instead installing on Kubernetes, there are no additional preparation steps required unless you are installing on Minikube. When installing on Minikube, ensure that you enable the Ingress add-on by executing the following command:
 
 [source,bash]
 ----
-bash <(curl -L http://git.io/getLatestKiali)
+minikube addons enable ingress
 ----
 
-By default, the installation script will prompt you to enter a username and passphrase for the credentials that you want users to enter in order to log in successfully to Kiali. The installation script can be customized through environment variables - see link:http://git.io/getLatestKiali[the comments at the top of the install script].
+When Kiali is installed in Minikube, this Ingress add-on will enable you to go directly to the Kiali UI at the URL `https://[minikube-ip]/kiali/console` where `[minikube-ip]` is what is reported when you run the command: `minikube ip`.
 
-Once the above has completed and the Docker image has been pulled from Dockerhub, go to the OpenShift console, select the istio-system project and determine the base-URL of Kiali
+=== Install Istio
 
-[#img-openshift]
-.OpenShift console
-image::os-console.png[OpenShift console]
-{nbsp} +
+Make sure your cluster environment has Istio installed. Because you are following these instructions to install the latest Kiali, you should not install the Kiali that comes with Istio (i.e. do not use the Helm setting `kiali.enabled=true` when installing Istio).
```

Comment:
Because you are following these instructions to install the latest version of Kiali, you should not install the Kiali version that is bundled with Istio (that is, do not use the Helm setting `kiali.enabled=true` when installing Istio).

Reply:
done (you should use the "insert suggestion" feature in github - its hard to figure out what you are changing and what is the same)

Reply:
I don't know about that feature.  *Googles*  "Available on Developer, Team, and Business Cloud plans"  Do we have one of those plans?